sway-error/src/error.rs

```diff
@@ -582,10 +582,10 @@ pub enum CompileError {
     BreakOutsideLoop { span: Span },
     #[error("\"continue\" used outside of a loop")]
     ContinueOutsideLoop { span: Span },
-    #[error("Configuration-time constant value is not a constant item.")]
-    ConfigTimeConstantNotAConstDecl { span: Span },
-    #[error("Configuration-time constant value is not a literal.")]
-    ConfigTimeConstantNotALiteral { span: Span },
+    #[error("Contract ID is not a constant item.")]
+    ContractIdConstantNotAConstDecl { span: Span },
+    #[error("Contract ID value is not a literal.")]
+    ContractIdValueNotALiteral { span: Span },
```

Comment:
Let's also add a small comment that says these will be removed in the future, linking to our issue.